OpenRA.Game/Graphics/SpriteRenderable.cs

```diff
@@ -17,30 +17,34 @@ namespace OpenRA.Graphics
 	public class SpriteRenderable : IPalettedRenderable, IModifyableRenderable, IFinalizedRenderable
 	{
 		public static readonly IEnumerable<IRenderable> None = new IRenderable[0];
-
```

Comment:
I think that newline can stay (that would also have avoided the rebase conflict).